pipeline/git/git.go

```diff
@@ -30,5 +34,7 @@ func (Pipe) Run(ctx *context.Context) (err error) {
 		PreviousTag: previous,
 		Diff:        log,
 	}
+	// removes usual `v` prefix
+	ctx.Version = strings.TrimPrefix(tag, "v")
```

Comment:
Yeah, I thought about that too... 

Seemed kind of OK for me to make it standard for everyone (ldflag, cask, fpm, yet-to-be-implemented stuff).